lib/core/src/packStruct.cpp

```diff
@@ -2028,8 +2028,9 @@ namespace
         if ( packTypeTable[typeInx].number == PACK_BIN_TYPE ) {
             /* bin type. need to decode */
             unsigned long outLen = len;
-            if ( int status = base64_decode( static_cast<const unsigned char *>(inPtr), inLen,
-                        static_cast<unsigned char *>(outPtr), &outLen ) ) {
+            if (int status = irods::base64_decode(
+                    static_cast<const unsigned char*>(inPtr), inLen, static_cast<unsigned char*>(outPtr), &outLen))
+            {
```

Comment:
Well, I quite hate this. :p

Reply:
Yeah. It's not pretty. PackStruct code isn't pretty either lol.

I'm fine with letting this ride though.